package.json

```diff
@@ -152,7 +152,7 @@
     "typescript": "^2.8.1"
   },
   "optionalDependencies": {
-    "electron-installer-debian": "^0.8.0",
+    "electron-installer-debian": "unindented/electron-installer-debian#transform-version-api",
```

Comment:
Not a fan of this, this can't be a package maintenance issue because it's on a branch in the main repo.  Can we wait for this to be released / do we have the ability to release it?

Reply:
Yeah, that's the # 1 reason why it's WIP :smile:

Reply:
We're having some trouble releasing a new `electron-installer-debian` version, so that's the only thing we're waiting on at this point.